### ISSUE TITLE
How to use your map generator

### ISSUE THREAD

**Issue by benjaminrobinet** [dcodeIO/RustWeb#10]
Hi, can you help me to generate a map from the seed and the map size with your code, but without a server ? ;)

**Comment by dcodeIO** [comment]
Tell me your seed and map size and I'll see what I can do :)

**Comment by benjaminrobinet** [comment]
Wow, fast answer, I have no seed and no size ^^ I'm just trying to understand your code to create an offline map generator (If you know what I mean ?)

**Comment by dcodeIO** [comment]
Well, the short answer is: It's not possible without running a plugin inside of an actual rust server.

**Comment by benjaminrobinet** [comment]
Someone seems to have greatfull make that ? http://www.playrusthq.com/generate-map

**Comment by benjaminrobinet** [comment]
And, he told me that he has used you code to do that. Did I havn't seen that we have to install your plugin ? That's possible

**Comment by dcodeIO** [comment]
He is probably using a wrapper around a rust server with the plugin installed. I assume he repeatedly starts a rust server with the given world size and seed, generates the image through issuing the respective rcon command and shuts it down again.

**Comment by benjaminrobinet** [comment]
Yeah it seems to be that, I havn't the time to make that. Thank you anyway for your answers!
